src/components/UserThumbnail.js

```diff
@@ -1,66 +1,84 @@
 import React from 'react';
 import {
-  StyleSheet,
-  View,
-  Text,
-  Image,
-  TouchableHighlight
+  StyleSheet, View, Text, Image, TouchableOpacity,
 } from 'react-native';
+import PropTypes from 'prop-types';
 
 import arrowIcon from '../assets/img/arrow_right.png';
 import DEFAULT_USER_AVATAR from '../assets/img/default_user_icon.png';
 import { SECONDARY_COLOR } from '../helpers/constants';
 
-export const UserThumbnail = ({ name, username, imageUrl }) => {
+export const UserThumbnail = ({
+  name, username, imageUrl, redirect,
+}) => {
   const avatarSource = imageUrl ? { uri: imageUrl } : DEFAULT_USER_AVATAR;
   return (
-    <View>
-      <TouchableHighlight>
-        <View style={styles.container}>
+    <View style={styles.container}>
+      <TouchableOpacity onPress={redirect}>
+        <View style={styles.content}>
           <Image style={styles.avatar} source={avatarSource} />
           <View style={styles.info}>
             <Text style={styles.mainText}>{name}</Text>
             <Text style={styles.subText}>{username}</Text>
           </View>
           <Image style={styles.arrow} source={arrowIcon} />
         </View>
-      </TouchableHighlight>
+      </TouchableOpacity>
       <View style={styles.hr} />
     </View>
   );
 };
 
+UserThumbnail.propTypes = {
+  name: PropTypes.string.isRequired,
+  username: PropTypes.string.isRequired,
+  imageUrl: PropTypes.string,
+  redirect: PropTypes.func,
+};
+
+// if a redirect url is not provided,
```

Comment:
remove all these comments, they should not make it into production.

Reply:
Oh. Alright. I would update that.